frontend/catalyst/qfunc.py

```diff
@@ -263,6 +263,8 @@ def wrap_single_shot_qnode(*_):
         results = catalyst.vmap(wrap_single_shot_qnode)(arg_vmap)
         if isinstance(results[0], tuple) and len(results) == 1:
             results = results[0]
-        return parse_native_mid_circuit_measurements(cpy_tape, aux_tapes, results, interface="jax")
+        return parse_native_mid_circuit_measurements(
+            cpy_tape, aux_tapes, results, postselect_mode="pad-invalid-samples"
+        )
```

Comment:
We use `"pad-invalid-samples"` unconditionally because the samples will already be valid/invalid depending on the chosen `postselect_mode`, so unconditionally using `"pad-invalid-samples"` here will just tell the processing function to replace invalid samples with fill-in values _iff_ there are any invalid samples present.